### DISCUSSION TITLE
Read Handler Slow to Respond Errors

### DISCUSSION THREAD

**Discussion by NeilJonesOnline** [RaresAil/homebridge-levoit-air-purifier#31] [Q&A] [answered]
Hi, and first many thanks for writing this plugin which I use in HOOBS to control two purifiers (a Core 300 and a Core 400).

Although both devices work fine in HomeKit through this plugin, I get a large amount of 'Read Handler Slow to Respond' errors in the logs (sample attached) for various outputs from the device. Any idea why this is? There doesn't seem to be any 'real' delay when controlling the devices, either in Home or in the native VeSync app, so I don't think there's a (significant) networking latency at play. Ping times are around 50-100ms.

Thanks,

Neil

[ Levoit Logs.txt](https://github.com/RaresAil/homebridge-levoit-air-purifier/files/8550511/Levoit.Logs.txt)

**Comment by RaresAil** [comment, marked as answer]
Hello, i didn't used Hoobs but, is the plugin running as a child bridge?

Edit: the platform must run as a child bridge

**Reply by NeilJonesOnline** [reply]
Hi, yes this plugin is running on its own dedicated bridge in HOOBS. I've updated to 1.1.13 but am still getting the same alerts in the logs.

**Reply by RaresAil** [reply]
What settings do you have for HomeBridge?

**Reply by RaresAil** [reply]
As there was no response i will mark it as completed

**Reply by almonde** [reply]
Just as a point of reference, I saw the same message in Homebridge until I installed it as a child bridge, once it was installed as a child bridge I haven't seen the message since.
